src/components/Pagination.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="ui pagination menu">
+    <a class="item" @click="setPage(currentPage - 1)">
+      Previous
+    </a>
+
+    <template v-if="currentPage <= 4">
```

Comment:
@hdong92 Surely I need to use templates. I don't want anything showing other than `<a>` tags.